### PR TITLE
Group name updated.

### PR DESCRIPTION
**Pull request by aravindan89**
### Change description ###
Group name updated

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by amitchidrewar1301 — APPROVED** [review]

File: key-vault.tf [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+  source = "git@github.com:hmcts/cnp-module-key-vault?ref=master"
   name = join("-", [var.product, var.env])
   product = var.product
   env = var.env
   tenant_id = var.tenant_id
   object_id = var.jenkins_AAD_objectId
   resource_group_name = azurerm_resource_group.rg.name
   # group id of dcd_reform_dev_azure
-  product_group_object_id = "56679aaa-b343-472a-bb46-58bbbfde9c3d"
+  product_group_name = "dcd_reform_dev_azure"
```
Comment (timja) [inline key-vault.tf:10]: don't you want to swap to your team group? not this group, people shouldn't be in this group any more, (the replacement group is 'DTS CFT Developers')

Comment (aravindan89) [reply key-vault.tf:10]: Hi @timja - Can we swap to this group **dcd_group_fees&pay_v2**?

Comment (timja) [reply key-vault.tf:10]: yes you can change to that, that would be my recommendation, no other changes

Comment (aravindan89) [reply key-vault.tf]: Thanks @timja. I have updated the group. Can you please approve ?

**Review by amitchidrewar1301 — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Pull request merged**